### PR TITLE
Fixes 2 minor map popup bugs

### PR DESCRIPTION
**Pull request by ivan-1kh**
Popup position now displays correctly.
Opening multiple popups now correctly toggle each other (only 1 can be open).
rel #4

### CODE REVIEW

File: src/pages/Home/components/PopupMarker/index.jsx [hunk 1]
```diff
@@ -18,8 +18,9 @@ export default ({ lat, lng, business, history, setCurrentMarker, currentMarker }
             {currentMarker === business.id && (
                 <InfoBox
                     position={{
-                        lat: lat,
-                        lng: lng - 0.02222,
+                        lat,
+                        lng,
+                        // lng: lng - 0.02222, Causes problems especially zooming in and out while popup is open
```
Comment (Karyum) [inline src/pages/Home/components/PopupMarker/index.jsx:22]: but the same problem for zooming in and out exists without  the `- 0.02222`. i want the the popup to appear under the marker. not to the right of it.
phones have more space vertically than horizontally.

File: src/pages/Home/components/PopupMarker/style.scss [hunk 2]
```diff
@@ -76,8 +76,7 @@
 
   button:hover {
     background-color: #508991;
-    font-weight: 800;
-    cursor: pointer;
```
Comment (Karyum) [inline src/pages/Home/components/PopupMarker/style.scss:80]: is it `cursor: pointer` by default then ?

**Review by Karyum — CHANGES_REQUESTED** [review]

File: src/pages/Home/components/PopupMarker/style.scss (hunk 2 above)
Comment (ivan-1kh) [reply src/pages/Home/components/PopupMarker/style.scss:80]: Yes.

File: src/pages/Home/components/PopupMarker/index.jsx (hunk 1 above)
Comment (ivan-1kh) [reply src/pages/Home/components/PopupMarker/index.jsx:22]: No, that problem is fixed when you don't play with the coordinates, but the popup remains to the bot-right of the selection.

If it is that necessary, the popup can be in the center by replacing InfoBox with InfoWindow, but creating a custom InfoWindow popup is very limited, as in our div will have to be inside the pre-made window.

image

I can make it this way if it is to your satisfaction, Senpai Mario.

Comment (Karyum) [reply src/pages/Home/components/PopupMarker/index.jsx:22]: there is a lot of dead space in that popup. remove the border and minimize the dead space

Comment (ivan-1kh) [reply src/pages/Home/components/PopupMarker/index.jsx:22]: I can't do a lot about this dead space, as you can see this is the area I can control:

image

Comment (Karyum) [reply src/pages/Home/components/PopupMarker/index.jsx:22]: Peek 2020-03-26 18-47
okay then just fix this

Comment (ivan-1kh) [reply src/pages/Home/components/PopupMarker/index.jsx:22]: Strange... how did you trigger this bug?

Comment (Karyum) [reply src/pages/Home/components/PopupMarker/index.jsx:22]: i just opened it chrome dev tools

Comment (Karyum) [reply src/pages/Home/components/PopupMarker/index.jsx:22]: i tried refreshing but still the same

Comment (ivan-1kh) [reply src/pages/Home/components/PopupMarker/index.jsx:22]: Gave it a redesign, now it should work on all screens dynamically.

Comment (Karyum) [reply src/pages/Home/components/PopupMarker/index.jsx:22]: again it's way too big
Screenshot_2020-03-27-11-18-55
this is on my phone, if you want to test the app on your mobile (i mena if you haven't figured it out already) change the API_URL to the heroku one and run the server, and then go to you laptop's port 3000 in your mobile

**Review by Karyum — APPROVED** [review]

**Pull request merged**